scripts/dts/devicetree.py

```diff
@@ -152,6 +160,9 @@ def parse_node(line, fd):
         'props': {},
         'name': build_node_name(name, addr)
     }
+    if alt_label:
+        node['alt_name'] = alt_label
```

Comment:
I find the mix of `alt_name` and `alt_label` mildly confusing, can't we use the same title for both?

Reply:
For 'alt_label', I wanted to provide an equivalent to 'label' in this script.
Thought, for use in extract_dts_includes, I preferred the use of alt_name not to mix with 'label' which is an often given property. In extract_dts_includes alias is not an alternative to 'label'.